packages/common/src/ui/notifications/AlertModal.tsx

```diff
@@ -17,55 +18,29 @@ export const AlertModal: React.FC<AlertModalProps> = ({
   title,
 }) => {
   return (
-    <Modal
-      open={open}
-      closeAfterTransition
-      BackdropComponent={Backdrop}
-      BackdropProps={{
-        timeout: 500,
-      }}
-    >
-      <Fade in={open}>
-        <Paper
-          sx={{
-            bgcolor: 'background.paper',
-            borderRadius: '16px',
-            boxShadow: theme => theme.shadows[7],
-            left: '50%',
-            p: 4,
-            position: 'absolute',
-            top: '50%',
-            transform: 'translate(-50%, -50%)',
-            width: 400,
-            '&:focus': {
-              outline: 'none',
-            },
-          }}
-        >
-          <Grid container gap={1} flexDirection="column">
-            <Grid container gap={1}>
-              <Grid item>
-                <AlertIcon color="primary" />
-              </Grid>
-              <Grid item>
-                <Typography
-                  id="transition-modal-title"
-                  variant="h6"
-                  component="span"
-                >
-                  {title}
-                </Typography>
-              </Grid>
-            </Grid>
-            <Grid item>
-              <Typography>{message}</Typography>
-            </Grid>
-            <Grid item display="flex" justifyContent="flex-end" flex={1}>
-              <DialogButton variant="ok" onClick={onOk} />
-            </Grid>
+    <BasicModal open={open} width={400} height={150}>
```

Comment:
uhh I removed the props and then when I did the confirmation modal I used the `BasicModal` which seems to do what we need.. looks the same to me.. but it likely isn't and I can revert

Reply:
looks fine to me!